LibJS+LibWeb: Spin the event loop until await has completed

davidot opened the pull request:
This fixes https://linus.dev crashing for me (although the github badges still don't seem to load :disappointed:)

(Sorry Linus!)

Technically this change is in LibJS, but it is really for LibWeb so I wasn't sure what to pick

linusg reviewed (approved):
LGTM! Thanks for looking into this :^)

Let's drop the LibWeb prefix and clarify in the commit message body instead.

linusg commented:
> (Sorry Linus!)

![](https://c.tenor.com/5Z-lf2IeYuUAAAAd/i-understood-that-reference-chris-evans.gif)

davidot commented:
> LGTM! Thanks for looking into this :^)
> 
> Let's drop the LibWeb prefix and clarify in the commit message body instead.

Removed LibWeb prefix and added a sentence about it.
